tests/vm_test.go

```diff
@@ -157,6 +157,23 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 
 			Expect(vm.Spec.Template.Spec.Domain.Machine.Type).To(Equal(explicitMachineType))
 		})
+
+		// this can be removed when the spec.config.MachineType field is finally removed
+		It("[Serial] should still honor the deprecated config.MachineType field", Serial, func() {
```

Comment:
Do we need both VM and VMI test? I believe this could be one of those "could be unit test". The change you make is pretty contained.

Reply:
Removed e2e tests and added a unit test, thanks for the suggestion!